### ISSUE TITLE
Conscript can't find repository

### ISSUE THREAD

**Issue by delfick** [n8han/conscript#77]
Hello,

I'm trying to install giter8 with cs and it complains he can't find the repository.

After some inspection, I believe the issue is https://github.com/n8han/conscript/blob/master/src/main/scala/github.scala#L57

My understanding is that this is trying to retrieve https://api.github.com/repos/n8han/giter8/master

But it should be getting https://api.github.com/repos/n8han/giter8/branches/master instead.

Can this be resolved please?

Thankyou

**Comment by MrOutis** [comment]
Did you use ```cs n8han/giter8``` ? I tried to install it a moment ago and I didn't encounter any problem.

**Comment by delfick** [comment]
Yeap, I use exactly that.

Interestingly, I just tried it on my computer and it worked.

So after some investigation, I've worked out what was causing the problem :)

I was running this inside a docker container and had translated the setup.sh into docker commands so that I don't have to rely on github being up to install conscript.

What I did wrong however was mess up the escaping of the quotes in the "cs" script.

This is fixed now and giter8 works once more!

Sorry for the noise.
